dask_cloudprovider/providers/aws/ecs.py

```diff
@@ -851,10 +867,24 @@ async def _create_cluster(self):
             self._cluster_name_template
         )
         self.cluster_name = self.cluster_name.format(uuid=str(uuid.uuid4())[:10])
+        create_cluster_kwargs = {
+            "clusterName": self.cluster_name,
+            "tags": dict_to_aws(self.tags),
+        }
+        if self._fargate_spot:
+            spot_cluster_kwargs = {
+                "capacityProviders": ["FARGATE", "FARGATE_SPOT"],
+                "defaultCapacityProviderStrategy": [
+                    {
+                        "capacityProvider": "FARGATE_SPOT",
+                        "weight": self._fargate_spot_weight,
+                    },
+                    {"capacityProvider": "FARGATE", "weight": 1},
+                ],
+            }
+            create_cluster_kwargs = {**create_cluster_kwargs, **spot_cluster_kwargs}
         async with self._client("ecs") as ecs:
-            response = await ecs.create_cluster(
-                clusterName=self.cluster_name, tags=dict_to_aws(self.tags)
-            )
+            response = await ecs.create_cluster(**create_cluster_kwargs)
```

Comment:
Why have tags been removed here?

Reply:
Shouldn't be, line 852, tags and cluster name are part of the initial `create_cluster_kwargs`

Reply:
Ah great no problem!